rules-tests/CodeQuality/Rector/ClassConstFetch/ConvertStaticPrivateConstantToSelfRector/Fixture/private-constant-with-static-should-use-self.php.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Utils\Rector\Tests\Rector\UseDateTimeImmutableRector\Fixture;
+
+final class Foo
```

Comment:
New fixture with `@final` docblock with non-native final class is needed.

Reply:
Added